src/main/java/seedu/address/model/residence/Residence.java

```diff
@@ -85,7 +85,7 @@ public boolean hasBooking(Booking booking) {
     }
 
     /**
-     * Returns a modified {@code Residence} with new booking.
+     * Returns a modified {@code Residence} with new {@code booking} added to it's BookingList.
```

Comment:
@code Booking*

Reply:
oh I was trying to refer to the parameter booking instead but not really sure where to use @code and when not to to be honest...

Reply:
i think it deals with how it renders the javadocs in html format, not too sure how it will look like when it renders though.